nrf70_bm_lib/docs/source/nrf70_bm_porting_guide.rst

```diff
@@ -89,6 +89,38 @@ For more information, visit the `nRF Connect SDK nrfxlib repository <https://git
 
    The git submodule pulls in the entire repo, but only `nrf_wifi` directory is required for the BM driver.
 
+Transmit (TX) power configuration
+*********************************
+
+The maximum transmit power achieved on an nRF70 Series device-based product depends on the frequency band and operating channel.
+This varies across different Printed Circuit Board (PCB) designs.
+
+Multiple calibrations and checks are implemented to ensure consistent TX power across channels and devices.
+However, the TX power ceilings depend on PCB design, which may result in Error Vector Magnitude (EVM) and spectral mask failures.
```

Comment:
This is still a bit confusing to me: is it really the ceilings that depend on the PCB design, or the achievable TX output power?

Reply:
I guess it is the second, hence my suggestion below. If I am wrong, just ignore the suggestion.

Reply:
Please note this text is taken as is from NCS https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/drivers/wifi/nrf70_native.html#devicetree_specification_configuration